server/src/main/java/umm3601/Server.java

```diff
@@ -10,6 +10,12 @@
 public class Server {
     public static void main(String[] args) throws IOException {
 
+        // This users looks in the folder `public` for the static web artifacts,
+        // which includes all the HTML, CSS, and JS files generated by the Angular
+        // build. This `public` directory _must_ be somwhere in the classpath;
+        // a problem which is resolved in `server/build.gradle`.
+        staticFiles.location("/public");
```

Comment:
Well, given that you've added the static file information *before* the redirect, it's likely catching the requests first. (routes are matched in the order they are defined!). As for why it works in development, I'd guess that it handles the static files "not existing" in a reasonable way...? Do you see any kind of log message about it not being able to find them?